src/sentry/static/sentry/app/components/forms/selectAsyncField.jsx

```diff
@@ -6,7 +6,8 @@ import SelectField from './selectField';
 
 class SelectAsyncField extends SelectField {
   static propTypes = {
-    ...SelectField.propTypes,
+    // TODO(ts)
+    // ...SelectField.propTypes,
```

Comment:
Leaving these as comments so we know how to write our `Props` type when we get to it.